<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowChat</class>
 <widget class="QMainWindow" name="MainWindowChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #add8e6;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>481</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>479</width>
       <height>409</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="textEditChat">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #ffffff;
border: 1px solid #ccc;
padding: 10px;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEditMensagem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border: 1px soolid #ccc;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEnviar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0074b8;
color: #ffffff;
border: none;
border-radius: 5px;
padding: 5px 10px</string>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
